features/search_across_libraries.feature

```diff
@@ -0,0 +1,12 @@
+Feature: Search across all libraries
+  As a researcher, I want to be able to search across all libraries
+  so that I don't need to know the specific focus of a particular library.
```

Comment:
@barnabyalter I see that for cucumber tests you only index 2 EADs:
https://github.com/NYULibraries/findingaids/blob/development/features/support/local_env.rb#L29
(also there is an error in path for the second one)
could I add photos_114.xml to the indexer ?



On Tue, Jan 20, 2015 at 4:41 PM, barnabyalter <notifications@github.com>
wrote:

> In features/search_across_libraries.feature
> <https://github.com/NYULibraries/findingaids/pull/27#discussion-diff-23258505>
> :
>
> > @@ -0,0 +1,12 @@
> > +Feature: Search across all libraries
> > +  As a researcher, I want to be able to search across all libraries
> > +  so that I don't need to know the specific focus of a particular library.
>
> @ekate <https://github.com/ekate> I don't think the feature here is the
> "All Collections" part, I think it's the "Library search field"
>
> Feature: Library search field
>   In order to search within specific Library collections or across all Libraries
>   As a researcher
>   I want to have a search field that allows me to limit by Library.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/NYULibraries/findingaids/pull/27/files#r23258505>.
>



-- 
Ekaterina Pechekhonova
Digital Library Programmer/Analyst
New York University
Libraries
email: ekate@nyu.edu
phone: 212-992-9993